safe-global/web-core
# chore: ugprade onboard

## What it solves

Removes reliance on onboard hotfix

## How this PR fixes it

The [official release](https://github.com/blocknative/web3-onboard/releases/tag/2.12.0) that resolves the `joi` now replaces the hotfix.

## How to test it

Connect a wallet and observe that transacting is still possible, as well as pairing.
